code/tools/vision/include/projection/projection.hpp

```diff
@@ -29,7 +29,7 @@
 #include "opencv2/video/tracking.hpp"
 #include "opencv2/imgproc/imgproc.hpp"
 #include "opencv2/highgui/highgui.hpp"
-
+#include "projection/InversePerspectiveMapping.hpp"
```

Comment:
Always put vertical space around different types of content, e.g. between include statements and namespace declarations